web-console/future-openshift/operators-overview/operators-overview.md

```diff
@@ -3,16 +3,15 @@
 This documentation details how operators will be surfaced on the Status page. Rather than exposing CSV objects, the status page should now expose the CRs being managed by identified CSVs in designated groups.
```

Comment:
Should this be updated a bit to explain that this is within the workloads tab of any Project Details view?

Reply:
Yes! Good catch :)